src/main/java/org/cryptomator/linux/quickaccess/NautilusBookmarks.java

```diff
@@ -0,0 +1,93 @@
+package org.cryptomator.linux.quickaccess;
+
+import org.cryptomator.integrations.common.CheckAvailability;
+import org.cryptomator.integrations.common.Priority;
+import org.cryptomator.integrations.quickaccess.QuickAccessService;
+import org.cryptomator.integrations.quickaccess.QuickAccessServiceException;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.StandardOpenOption;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+
+@Priority(100)
+@CheckAvailability
```

Comment:
Also add `@OperatingSystem(OperatingSystem.Value.LINUX)`, as it filters faster than invoking `isSupported()`

Reply:
Done in dbf2826957d1be163f6c99801519d06259573580